keyboards/splitkb/elora/rev1/config.h

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2023 splitkb.com <support@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// Make it easier to enter the bootloader
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+// I2C0, onboard SSD1306 socket and I2C to Myriad module
+#define I2C_DRIVER   I2CD0
+#define I2C1_SDA_PIN GP0
+#define I2C1_SCL_PIN GP1
+// We need to slow down the I2C clock because the default of 400.000
+// fails to communicate with Zetta ZD24C02A EEPROM on a Myriad card.
+// This somehow results in a clock of 127kHz, but that's okay
+#define I2C1_CLOCK_SPEED 100000
```

Comment:
Yeah, I think we discussed that on the Discord a while ago already! We don't _really_ care about the exact frequency, as both 100kHz and 127kHz will work just fine.

I wanted to document the discrepancy just in case anyone had to look into it in the future, but it seems that last line can now be removed. :smiley: